step-capstone/src/components/TravelObjects/DraggableUnfinalize.js

```diff
@@ -0,0 +1,55 @@
+import React, {useState} from "react";
+import { useDrag } from "react-dnd";
+import TravelObject from "./TravelObject";
+import _, { clone } from "lodash";
+import { ItemTypes } from "../../scripts/DragTravelObject";
+
+
+export default function DraggableUnfinalized(props) {
```

Comment:
Nit: The function is `DraggableUnfinalized` but the file is named `DraggableUnfinalize`, please make the names match as in other files.

Reply:
I have renamed the file. Thanks!